src/main/java/co/grandcircus/FinalProject/FinalController.java

```diff
@@ -522,68 +524,55 @@ public String packBudgetBin(@RequestParam Double budget, Model model) {
 				}
 			}
 		}
-		
+
 		List<ListContainer> listOfItemLists = new ArrayList<>();
-		Collections.sort(wishes, new Comparator<WishList>() { //sort the array on price so that the lowest price is always first.  This should guaruntee that a best fit is found for the budget.
-		    @Override
-		    public int compare(WishList c1, WishList c2) {
-		        return Double.compare(c1.getPrice(), c2.getPrice());
-		    }
+		Collections.sort(wishes, new Comparator<WishList>() { // sort the array on price so that the lowest price is
+																// always first. This should guaruntee that a best fit
+																// is found for the budget.
+			@Override
+			public int compare(WishList c1, WishList c2) {
+				return Double.compare(c1.getPrice(), c2.getPrice());
+			}
 		});
-		
 
-				
-				Set<WishList> bin = new HashSet<>();
-				Double tempbudget = budget;
-				
-				if(tempbudget >= wishes.get(0).getPrice()) {
-					bin.add(wishes.get(0));
-					Double price = wishes.get(0).getPrice();
-					tempbudget = tempbudget-price;
-				
-				while(price <= tempbudget) {
-					
-					for(int j=1; j < wishes.size(); j++) {
-					
-							if(tempbudget >= wishes.get(j).getPrice()) {
-								
-								bin.add(wishes.get(j));//add wish to shopping cart
-								price = wishes.get(j).getPrice();
-								tempbudget = tempbudget-price;
-								
-							} else {
-							
-							}
-					}	
-				ListContainer binObject = new ListContainer();
-				binObject.setWishlists(bin);
-				binObject.setLength(bin.size());
-				
-				listOfItemLists.add(binObject);
-				
-			}	
-		}
-		
-		
-		model.addAttribute("user",user);
-		model.addAttribute("budget",budget);
-		
-			
+		Set<WishList> bin = new HashSet<>();
 		
-		model.addAttribute("listoflists",listOfItemLists.get(0)); //we may have to set this to only pick one or two lists, otherwise the runtime can get excessive.
+//		<<===== Start Sean B Changes =====>>
+		Double price = 0.0;
+		for (int j = 0; j < wishes.size(); j++) {
+			if (budget >= price + wishes.get(j).getPrice()) {
+				bin.add(wishes.get(j));// add wish to shopping cart
+				price += wishes.get(j).getPrice();
+			}
+		}
+		//You may want to put some handling for empty lists if you haven't already.
+		//In addition, this is formatted to deal with the list you have inputed where the lowest price comes in first.
+		//If you ordered it High to low, it would possibly give you a better result.
+//		<<====== End Sean B Changes ======>>
```

Comment:
If you do decide to order high to low:
```suggestion
//		<<===== Start Sean B Changes =====>>
		Double price = 0.0;
		for (int j = 0; j < wishes.size(); j++) {
			if (budget >= price + wishes.get(j).getPrice()) {
				bin.add(wishes.get(j));// add wish to shopping cart
				price += wishes.get(j).getPrice();
			} else break;
		}
		//You may want to put some handling for empty lists if you haven't already.
		//In addition, this is formatted to deal with the list you have inputed where the lowest price comes in first.
		//If you ordered it High to low, it would possibly give you a better result.
//		<<====== End Sean B Changes ======>>
```